src/manufacturer/manufacturerLandingPage.component.tsx

```diff
@@ -63,66 +63,67 @@ function ManufacturerLandingPage() {
         </Button>
       </Grid>
       {manufacturerData && (
-        <Grid item xs={12}>
-          <Grid container spacing={1} flexDirection="column">
-            <Grid item xs={12}>
-              <Typography sx={{ margin: 1, textAlign: 'center' }} variant="h4">
-                {manufacturerData.name}
+        <Grid
+          container
+          spacing={1}
+          sx={{ px: '192px' }}
+          alignContent={'center'}
+          flexDirection="row"
+        >
+          <Grid item xs={12}>
```

Comment:
- can you change the non-header text to to use color={'text.secondary'} , for the url, telephone number and address. 
<br>
EXAMPLE:

```
  <Grid item xs={12} my={2}>
            <Typography textAlign={'center'} variant="h6">
              Telephone number:
            </Typography>
          </Grid>
          <Grid item xs={12}>
            <Typography sx={{ mx: '8px' }} textAlign={'center'} variant="body1" color="text.secondary">
              {manufacturerData.telephone}
            </Typography>
          </Grid>
```
<br>

- can you also default the values to 'None' if they don't exist on the manufacturers landing page.

<br>

- add the manufacturer telephone number to the catalogue item landing page and catalogue details panel.